File: TransCoder_tokenized_test_set_functions/cpp/FIND_PAIRS_IN_ARRAY_WHOSE_SUMS_ALREADY_EXIST_IN_ARRAY_1.cpp
void findPair ( int arr [ ], int n ) {
  unordered_set < int > s;
  for ( int i = 0;
  i < n;
  i ++ ) s . insert ( arr [ i ] );
  bool found = false;
  for ( int i = 0;
  i < n;
  i ++ ) {
    for ( int j = i + 1;
    j < n;
    j ++ ) {
      if ( s . find ( arr [ i ] + arr [ j ] ) != s . end ( ) ) {
        cout << arr [ i ] << " " << arr [ j ] << endl;
        found = true;
      }
    }
  }
  if ( found == false ) cout << "Not exist" << endl;
}

File: TransCoder_tokenized_test_set_functions/python/FIND_PAIRS_IN_ARRAY_WHOSE_SUMS_ALREADY_EXIST_IN_ARRAY_1.py
def findPair ( arr, n ) :
    s = {
    i : 1 for i in arr }
    
    found = False
    for i in range ( n ) :
        for j in range ( i + 1, n ) :
            if arr [ i ] + arr [ j ] in s . keys ( ) :
                print ( arr [ i ], arr [ j ] )
                found = True
    if found == False :
        print ( "Not exist" )
    
